dts/bindings/ipm/intel,sedi-ipm.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2020-2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: INTEL SEDI IPM controller.
+
+compatible: "intel,sedi-ipm"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  peripheral-id:
+    type: int
+    required: true
+    description: sedi instance id of ipm
+
+  timeout-ms:
+    type: int
+    required: true
+    description: send timeout in ms
```

Comment:
Is this timeout really more of a hardware property and not a software configuration detail? If it's more of a software thing, then it would be more suited as a Kconfig option.

Reply:
removed this field, and make a macro defination on driver